project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import sbt._
 
 object Dependencies {
   lazy val awsVersion = "1.11.678"
-  lazy val atomLibVersion = "1.4.0"
+  lazy val atomLibVersion = "2.0.0-PREVIEW.fix-tests-under-java-17.2024-01-18T1039.b4d55b3d"
```

Comment:
This is a preview build provided of https://github.com/guardian/atom-maker/pull/94, which adds support for Play 2.9 - once that PR is merged, we should update _this_ PR to use the final 2.0.0 version, but in the mean time this is good for trying out the changes.